components/form/pta/src/index.js

```diff
@@ -16,11 +16,17 @@ class FormPta extends Component {
   constructor(props) {
     super(props)
 
-    const {formUrl: BASE_URL, onSubmit, onError, eventListeners, ...settings} = this.props
+    const {
```

Comment:
this changes shouldn't be part of this PR, right?

Reply:
In this case it shouldn't be a big deal but you are changing form/pta stuff under the card/arrow scope so no release changes will be found

Reply:
ok, being very controlled changes, and not needing to make a new release of the form/pta, I upload it and in a future modification of the form/pta we solve it.
Thank you very much!